app/views/compute_resources_vms/form/libvirt/_network.html.erb

```diff
@@ -1,36 +1,30 @@
-<div class="fields">
-  <div class="form-group">
-  <% nat    = compute_resource.networks %>
-  <% bridge = compute_resource.interfaces %>
 
-  <%= f.fields_for 'compute_attributes', OpenStruct.new(f.object.compute_attributes) do |f| %>
+<% nat    = compute_resource.networks %>
+<% bridge = compute_resource.interfaces %>
 
-    <%=
-    selectable_f f, :type, libvirt_networks(compute_resource), {},
-                     { :label    => _('Network type'),
-                       :disabled => disabled,
-                       :onchange => 'libvirt_network_selected(this)' } %>
+<%=
```

Comment:
I see it's not your code but could you put `<%=` on same line with `selectable_f` please?